examples/launch_height_sagemaker.py

```diff
@@ -43,7 +43,7 @@
     metric = "mean_loss"
 
     # Random search without stopping
-    scheduler = FIFOScheduler(
+    scheduler = RandomSearch(
         config_space,
         searcher='random',
```

Comment:
You can remove the searcher arg then, right?

Reply:
Good catch, thanks.